### PR TITLE
Aded new product page and restructured admin panel layout

### LINKED ISSUE #357: Create new product route
Create new product route and page (page without content)
I guess it will also require changing current tabs implementation

### PR DESCRIPTION
**Pull request by heshvab**
- Instead of query pamateres, I used simple pages with layout which includes menu from the left side.
- Instead of OrdersTab, ProductsTab and UsersTab added corresponding pages.
- Added DashboardLayout.
- Since tabs became separate pages I also changed structure of admin panel files in our codebase. Layout and its components are located In layouts. I moved orders filter drawer and its components in containers folder. 
- Added new product page

### CODE REVIEW

**Review by mhevyk — COMMENTED** [review]
Visually it looks amazing! I noticed that some disable-like styles are applied to tabs. I suggest removing them

image

File: code/src/constants/routes.ts [hunk 1]
```diff
@@ -21,7 +21,23 @@ const routes = {
...
+      relativePath: "products",
+      new: {
+        path: "/dashboard/products/new",
+        relativePath: "products/new"
+      }
+    },
+    users: {
+      path: "/dashboard/users",
+      relativePath: "users"
+    }
```
Comment (mhevyk) [inline code/src/constants/routes.ts]: Excellent job! I am curious about `relativePath` keys here. I think they can be kinda inferred from path by just joining paths. We can declare only essential part of url and maybe create util to create absolute or relative paths. What do you think about it?

**Review by mhevyk — COMMENTED** [review]
Also I did not notice tests for dashboard products and dashboard-new-products page. You did not write them because you are working on these tabs, am I correct?

**Comment by heshvab** [comment]
> Also I did not notice tests for dashboard products and dashboard-new-products page. You did not write them because you are working on these tabs, am I correct?

Yeap, that is correct. There are no much sense of writing tests without ready implementation like fetching products

File: code/src/constants/routes.ts (hunk 1 above)
Comment (heorhiiss) [reply code/src/constants/routes.ts]: Thanks for suggestion. I found out that I don't really need relative paths in this case, so I removed all them

**Comment by heshvab** [comment]
> Visually it looks amazing! I noticed that some disable-like styles are applied to tabs. I suggest removing them
> 
> image

Thanks. It was because AppLink setted some styles when the link is active. Fixed

**Review by mhevyk — APPROVED** [review]

**Review by D1nviz — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Pull request merged**